web/index.html

```diff
@@ -17,237 +17,91 @@
     <div id="map"></div>
     <script>
       var map;
-      var marker;
-      var markers = {};
-      var loc = {}, cachedLoc = {};
+      var tMarker;
+      var cMarker;
+      var tLoc = {}, cachedtLoc = {};
+      var cLoc = {}, cachedcLoc = {};
       var inventory = {};
       var playerInfo = {};
       var pokedex = {};
       var bagPokemon = {};
       var bagItems = {};
       var bagCandy = {};
       var emptyDex = [];
-      var div = document.createElement('div');
+      var cInfo = {};
+      var catchable = {};
+      var pokemonArray = {};
       function initMap() {
         map = new google.maps.Map(document.getElementById('map'), {
           center: {lat: 50.0830986, lng: 6.7613762},
           zoom: 8
         });
       setTimeout(function(){
         placeTrainer();
+        addCatchable();
         updateInventory();
         setTimeout(function(){
           setInterval(updateTrainer, 1000);
+          setInterval(addCatchable, 1000);
         }, 5000);
       }, 5000);
       }
       function placeTrainer() {
-        loadJSON('location.json?_=' + new Date().getTime(),
-          function(data) { loc = data; map.setZoom(16); map.panTo({lat: parseFloat(loc.lat), lng: parseFloat(loc.lng)}); },
+        loadJSON('location.json',
+          function(data) { tLoc = data; map.setZoom(16); map.panTo({lat: parseFloat(tLoc.lat), lng: parseFloat(tLoc.lng)}); },
```

Comment:
I am going to bed.  You guys can figure out whether you want to merge this I guess.